src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+
+export default function calculate(
+  data = {
```

Comment:
```({total ,next, operation}, buttonName )``` you can access the total, next and operation in the code afterwards
or
```(data, buttonName)``` the restructure next, total and operation in the case.
I think this way is more cleaner than what we have now